lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -549,3 +563,29 @@ MITIGATION_02_CRI_DCMAW:
       targetState: PYI_ANOTHER_WAY
```

Comment:
In case of error from DCMAW during mitigation the flow should take user to MITIGATION_02_SUGGEST_F2F page. So you need to handle this for all other error events from DCMAW.